### PR TITLE
Pin pyglet version to the latest compatible with gym 0.12.4

### LINKED ISSUE #768: Nightly builds are failing
https://travis-ci.com/rlworkgroup/garage/jobs/213465246

I think it may have been a API change in pyglet. we may need to pin that dep.

### PR DESCRIPTION
**Pull request by gitanshu**
Closes #768

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
     'mujoco-py<2.1,>=2.0',
     'numpy==1.14.5',
     'psutil',
+    'pyglet<1.4.0,>=1.3.0',
```
Comment (ryanjulian) [inline setup.py]: can you leave a comment about why this is pinned?

**Comment by krzentner** [comment]
I'm confused. The changes on this branch do a few things, but don't actually seem to pin pyglet. Did I miss something?

**Comment by gitanshu** [comment]
> I'm confused. The changes on this branch do a few things, but don't actually seem to pin pyglet. Did I miss something?

It started with pinning the pyglet version, then Ryan found that gym started pinning the pyglet version in the latest release, so he switched to that. But gym also updated atari-py in 0.13.x, and so now when we do spec.make() on atari envs, it hangs up at a [native call here](https://github.com/openai/atari-py/blob/7649e3891f45d99cab3174548bf59f2040a358a2/atari_py/ale_python_interface.py#L149).

So I'd say we rollback to pinning pyglet version for now. What do you think?

**Comment by ryanjulian** [comment]
I am so tired of fixing this bug.

@gitanshu after the release, can you send a PR to openai/gym fixing this?

**Comment by ryanjulian** [comment]
ok please roll this back to the original fix and merge it, and file an issue to update gym.

**Comment by ryanjulian** [comment]
great work, thanks for the triage.

**Pull request merged**